app/javascript/src/components/Dashboard/Contacts/Table.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+
+import { MenuHorizontal } from "neetoicons";
+import { Table as NeetoUITable, Avatar, Dropdown, Typography } from "neetoui";
+
+import { formatDate } from "../utils";
+
+const { Menu, MenuItem } = Dropdown;
+
+const renderAvatarWithNameAndRole = (
+  _,
+  { firstName, lastName, role, userAvatarUrl }
+) => (
+  <div className="flex space-x-2">
+    <Avatar size="large" user={{ imageUrl: userAvatarUrl }} />
+    <div className="flex flex-col">
+      <Typography className="neeto-ui-text-gray-800" style="h5">
+        {firstName} {lastName}
+      </Typography>
+      <Typography className="neeto-ui-text-gray-600" style="body3">
+        {role}
+      </Typography>
+    </div>
+  </div>
+);
+
+const renderDropdownMenu = () => (
+  <Dropdown buttonStyle="text" icon={() => <MenuHorizontal size={16} />}>
+    <Menu>
+      <MenuItem.Button>Edit</MenuItem.Button>
+      <MenuItem.Button style="danger">Delete</MenuItem.Button>
+    </Menu>
+  </Dropdown>
+);
```

Comment:
Let's move these to an `utils.jsx` file within the closest scope. Refer: https://academy.bigbinary.com/learn-react/react-best-practices/structuring-code

Reply:
Okay. I will make the necessary changes. I have some doubts:
 - Can I link the PR which implements the review comments to the same issue or do i need to create a separate issue for that?
 - Can I fix review comments for multiple PRs in a single PR?

Reply:
>Can I fix review comments for multiple PRs in a single PR?

This is fine.

>Can I link the PR which implements the review comments to the same issue or do i need to create a separate issue for that?

Raise a separate issue, add the links to the review comments in that issue and then make a PR fixing all those review comments.